index.js

```diff
@@ -759,3 +759,20 @@ is.hex = function (value) {
 is.symbol = function (value) {
   return typeof Symbol === 'function' && toStr.call(value) === '[object Symbol]' && typeof symbolValueOf.call(value) === 'symbol';
 };
+
+is.oneOf = function (value) {
+  for (var _len = arguments.length, types = Array(_len > 1 ? _len - 1 : 0), _key = 1; _key < _len; _key++) {
```

Comment:
variadic functions are both slower to work with and harder to reason about.

If this was to be added, it would require passing only two arguments, `value` and `types` - and if the second one wasn't an Array, it should throw an exception.

However, at that point, why not `types.some(function (type) { return is.function(is[type]) && is[type](value); })` in user code?